contrib/lib/Cargo.toml

```diff
@@ -61,8 +61,8 @@ r2d2 = { version = "0.8", optional = true }
 r2d2_postgres = { version = "0.14", optional = true }
 mysql = { version = "14", optional = true }
 r2d2_mysql = { version = "9", optional = true }
-rusqlite = { version = "0.14.0", optional = true }
-r2d2_sqlite = { version = "0.6", optional = true }
+rusqlite = { version = "0.15.0", optional = true }
```

Comment:
I'm fairly certain that this is a breaking change, which means we won't be able to ship it in `0.4.x`. Can we either 1) allow both versions, or 2) keep the old version?

Reply:
I'll investigate option 2, first. I'm sure we can keep the old version. Thanks for calling out the breaking change issue.